### ISSUE TITLE
Debug user discord image not always available.

### ISSUE THREAD

**Issue by AbdBarho** [LAION-AI/Open-Assistant#2196]
Users logged in with discord don't always have valid images.

My initial hypothesis:

We implemented a refresh of the user's profile images https://github.com/LAION-AI/Open-Assistant/pull/2043, which worked, but fails sometimes.

It seems that this is related to how nextauth handles credentials, or more precisely, how it does not correctly handles refresh tokens.

https://github.com/nextauthjs/next-auth/issues/3599

https://github.com/nextauthjs/next-auth/issues/3599#issuecomment-1091608216

TLDR: we might need to handle the refresh of the user token ourself.

**Comment by notmd** [comment]
This is fixed right?

**Comment by AbdBarho** [comment]
Unfortunately not.
